### PR TITLE
Profile system 

### PR DESCRIPTION
**Pull request by TeerapatTrepopsakulsin**
### Description

- Profile features
    + create profile
    + show profile
    + edit profile
- Profile pages
    + profile index `/profile` (profile:index): show only user's profile, if created,/create profile feature.
    + profile detail `/profile/<profile.id>` (profile:detail):  show user's profile according to parameter `profile.id`/ edit profile feature.

### Functional Acceptance Criteria

- Backend works:
    + Show, Create, Edit features works according to the request (tested)
    + has_profile data when `GET /profile` (tested)
    + User cannot create profile twice (tested)
    + Cannot edit other user profile (tested)

### Checklist 

- [x] I have performed a self-review of my code.
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation, if applicable.
- [x] My code follows the Coding Guidelines.

### Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Documentation update
- [ ] Other (please specify)

### Additional Information
- `/profile`
    + GET - show user profile and `has_profile` status
    + POST - create profile
- `/profile/<profile.id>`
    + GET - show profile according to `profile.id`
    + PUT - edit profile (only profile owner can)
- Don't forget to _migrate_

### CODE REVIEW

File: profiles/models.py
Comment (OmegaOoh) [inline profiles/models.py:1]: I said no DoB, unless you can justify it

File: profiles/serializer/model_serializers.py
Comment (OmegaOoh) [inline profiles/serializer/model_serializers.py:1]: Undescriptive Classdocstrings

File: profiles/serializer/permissions.py
Comment (OmegaOoh) [inline profiles/serializer/permissions.py:1]: Bad ClassName as we discussed on Code Review

**Review by OmegaOoh — CHANGES_REQUESTED** [review]
I saw some problem with it, please resolve before request for rereview

File: profiles/models.py
Comment (TeerapatTrepopsakulsin) [reply profiles/models.py:1]: - Some activities may involve age restriction (e.g. Bar)
- Birthday party, maybe

File: profiles/serializer/model_serializers.py
Comment (TeerapatTrepopsakulsin) [reply profiles/serializer/model_serializers.py:1]: Edited.

File: profiles/serializer/permissions.py
Comment (TeerapatTrepopsakulsin) [reply profiles/serializer/permissions.py:1]: Thanks for mentioning. Edited.

File: profiles/models.py
Comment (OmegaOoh) [reply profiles/models.py:1]: Not Approved. Please Delete it

**Comment by TeerapatTrepopsakulsin** [comment]
Last `makemigrations` create activities migration file (0005)

**Review by OmegaOoh — APPROVED** [review]
Good job. Approved

**Review by opxz7148 — APPROVED** [review]
Good job. User profile has useful information as we plan. _Approve_

**Pull request merged**